core/settings/Settings.py

```diff
@@ -155,3 +155,24 @@ class AppName(object):
     WITH_DASH = 'tns-app'
     WITH_SPACE = 'Test App'
     WITH_NUMBER = '123'
+
+
+class Paths(object):
```

Comment:
Can we move those paths to TnsPaths class (see products/nativescript/tns_paths.py)?